climada/hazard/centroids/centr.py

```diff
@@ -140,9 +142,9 @@ def check(self):
                 LOGGER.error('Missing meta information: width, height,'
                              'crs or transform')
                 raise ValueError
-            if self.meta['transform'][4] > 0:
-                LOGGER.error('Meta does not contain upper left corner data.')
-                raise ValueError
+            if self.meta['transform'][4] > 0 or self.meta['transform'][0] < 0:
+                LOGGER.warning('Meta does not contain upper left corner data.'
+                               'Some functionality might not work as expected.')
```

Comment:
This warning is way too fuzzy and will most probably lead to issues in the future.

Reply:
So do you suggest to raise an error or do you suggest do modify the output/explanation?

Reply:
The minimal solution is to provide a more detailed explanation, but is sub-optimal as the actual problem is not solved: why was there this constraint in the first place and do we need it?

Ideally, we want to know whether we need an error or whether the warning can be removed. I do not know the answer, however.

Reply:
Okay, then let's wait for answers in #100 before making decisions about this PR.